### PR TITLE
fix(ui5-color-picker): rtl support

### PR DESCRIPTION
**Pull request by Todor-ads**
Fixes: #4892

### CODE REVIEW

File: packages/main/src/themes/ColorPicker.css [hunk 1]
```diff
@@ -82,7 +82,7 @@
 
 [ui5-slider].ui5-color-picker-hue-slider::part(progress-container) {
     background-size: 100%;
-    background-image: -webkit-linear-gradient(left, #f00, #ff0, #0f0, #0ff, #00f, #f0f, #f00);
+    background-image: -webkit-linear-gradient(right, #f00, #ff0, #0f0, #0ff, #00f, #f0f, #f00);
```
Comment (a-z-ivanov) [inline packages/main/src/themes/ColorPicker.css:85]: why the change?

File: packages/main/src/ColorPicker.js [hunk 2]
```diff
@@ -256,23 +257,23 @@ class ColorPicker extends UI5Element {
 		alphaProgressSlider.style.width = "calc(100% + 11px)";
 		alphaProgressSlider.style.height = "18px";
 		alphaProgressSlider.style.position = "absolute";
-		alphaProgressSlider.style.marginTop = "-10px";
+		alphaProgressSlider.style.marginTop = this.effectiveDir === "rtl" ? "10px" : "-10px";
```
Comment (a-z-ivanov) [inline packages/main/src/ColorPicker.js:260]: are you sure the direction should change a top margin?

File: packages/main/src/ColorPicker.js [hunk 3]
```diff
@@ -227,7 +227,8 @@ class ColorPicker extends UI5Element {
 		const hueProgressSlider = hueSlider.querySelector(".ui5-slider-progress-container"),
 			hueHandle = hueSlider.querySelector(".ui5-slider-handle"),
 			alphaProgressSlider = alphaSlider.querySelector(".ui5-slider-progress-container"),
-			alphaHandle = alphaSlider.querySelector(".ui5-slider-handle");
+			alphaHandle = alphaSlider.querySelector(".ui5-slider-handle"),
+			alphaProgressSliderColor = this.effectiveDir === "rtl" ? "right" : "left";
```
[the remark this answers is not in the document]
Comment (Todor-ads) [reply packages/main/src/ColorPicker.js:231]: Done

File: packages/main/src/ColorPicker.js [hunk 4]
```diff
@@ -256,23 +257,23 @@ class ColorPicker extends UI5Element {
...
 		// ui5-slider.ui5-color-picker-hue-slider::part(progress-container)
 		hueProgressSlider.style.backgroundSize = "100%";
-		hueProgressSlider.style.backgroundImage = "-webkit-linear-gradient(left, #f00, #ff0, #0f0, #0ff, #00f, #f0f, #f00)";
-		hueProgressSlider.style.backgroundImage = "-moz-linear-gradient(left, #f00, #ff0, #0f0, #0ff, #00f, #f0f, #f00)";
-		hueProgressSlider.style.backgroundImage = "-ms-linear-gradient(left, #f00, #ff0, #0f0, #0ff, #00f, #f0f, #f00)";
-		hueProgressSlider.style.backgroundImage = "linear-gradient(left, #f00, #ff0, #0f0, #0ff, #00f, #f0f, #f00)";
+		hueProgressSlider.style.backgroundImage = `-webkit-linear-gradient(${alphaProgressSliderColor}, #f00, #ff0, #0f0, #0ff, #00f, #f0f, #f00)`;
+		hueProgressSlider.style.backgroundImage = `-moz-linear-gradient(${alphaProgressSliderColor}, #f00, #ff0, #0f0, #0ff, #00f, #f0f, #f00)`;
+		hueProgressSlider.style.backgroundImage = `-ms-linear-gradient(${alphaProgressSliderColor}, #f00, #ff0, #0f0, #0ff, #00f, #f0f, #f00)`;
+		hueProgressSlider.style.backgroundImage = `linear-gradient(${alphaProgressSliderColor}, #f00, #ff0, #0f0, #0ff, #00f, #f0f, #f00)`;
```
[the remark this answers is not in the document]
Comment (Todor-ads) [reply packages/main/src/ColorPicker.js:269]: done

File: packages/main/src/ColorPicker.js [hunk 5]
```diff
@@ -256,23 +257,23 @@ class ColorPicker extends UI5Element {
...
 		hueProgressSlider.style.backgroundColor = "none";
 
 		// ui5-slider.ui5-color-picker-alpha-slider::part(progress-container)
-		alphaProgressSlider.style.backgroundImage = "-webkit-linear-gradient(left, #fff, #979797)";
-		alphaProgressSlider.style.backgroundImage = "-moz-linear-gradient(left, #fff, #979797)";
-		alphaProgressSlider.style.backgroundImage = "-ms-linear-gradient(left, #fff, #979797)";
-		alphaProgressSlider.style.backgroundImage = "linear-gradient(left, #fff, #979797)";
+		alphaProgressSlider.style.backgroundImage = `-webkit-linear-gradient(${alphaProgressSliderColor}, #fff, #979797)`;
```
[the remark this answers is not in the document]
Comment (Todor-ads) [reply packages/main/src/ColorPicker.js:273]: done

**Review by a-z-ivanov — APPROVED** [review]
